src/holochainClient.js

```diff
@@ -108,7 +108,7 @@ let holochainClient
 async function initAndGetHolochainClient () {
   if (holochainClient) return holochainClient
   try {
-    let url = process.env.NODE_ENV === 'production' ? undefined : process.env.REACT_APP_DNA_INTERFACE_URL
+    let url = process.env.NODE_ENV === 'production' ? '/api/v1/ws/' : process.env.REACT_APP_DNA_INTERFACE_URL
```

Comment:
I was surprised by this change, so just want to double check. this is the base url that's passed to `hc-web-client`. My understanding is that we want to leave that undefined in production as the conductor handles that. Are you sure you want it to be this path?

Reply:
`connect` ([source](https://github.com/holochain/hc-web-client/blob/master/src/index.ts#L29)) needs to be passed a url at which conductor is listening. Our conductor happens to be running on the HPOS, so we need to provide a URL to HPOS. That's where nginx comes into play via [those settings](https://github.com/Holo-Host/holo-nixpkgs/blob/develop/profiles/logical/hpos/default.nix#L114) and redirects calls to the conductor instance running on HPOS.

Reply:
ok, got it. In that case it's good to merge as far as I'm concerned.